src/include/daos_srv/dtx_srv.h

```diff
@@ -205,7 +206,7 @@ enum dtx_flags {
 	DTX_FOR_MIGRATION	= (1 << 3),
 	/** Ignore other uncommitted DTXs. */
 	DTX_IGNORE_UNCOMMITTED	= (1 << 4),
-	/** Resent request. */
+	/** Resent request. Out-of-date. */
```

Comment:
just confirm what "Out-of-date" means no usage in future?
sorry don't quite understand why no longer need this flag after this patch?

Reply:
Nobody uses dth->dth_resent any longer. Related logic has become dead code for long time. This patch touches a lot of DTX logic for collective, then does some cleanup.